src/dragonball/src/api/v1/virtio_net.rs

```diff
@@ -68,6 +68,7 @@ pub struct VirtioConfig {
     pub allow_duplicate_mac: bool,
 }
 
+/// Config for vhost-user-net device
 #[cfg(feature = "vhost-user-net")]
```

Comment:
@justxuewei Hello~ I think `src/dragonball/src/api/v1/virtio_net.rs: line 72` should be removed. `VhostUserConfig` is gated behind the `vhost-user-net` feature, but is exported when other features are enabled.

Reply:
Thank you for reporting this. I've raised a PR to fix this. Please refer to https://github.com/kata-containers/kata-containers/pull/8744.